docs/src/ladder/classic-ladder.adoc

```diff
@@ -812,141 +666,107 @@ image::images/Config-io.png["Config I/O",align="center"]
+So register numbers 1-8 are mapped onto ClassicLadder's %B variables starting at %B1 and ending at %B8.
 
-The second line is set for 2 output bits starting at the ninth
-register number (register 9) so register numbers 9-10 are mapped onto
-ClassicLadder's %Q variables starting at %Q9 ending at %Q10.
+The second line is set for 2 output bits starting at the ninth register number (register 9) so register numbers 9-10 are mapped onto ClassicLadder's %Q variables starting at %Q9 ending at %Q10.
 
-The third line is set to write 2 registers (16 bits each) starting at
-the 0th register number (register 0) so register numbers 0-1 are
-mapped onto ClassicLadder's %W variables starting at %W0 ending at %W1.
+The third line is set to write 2 registers (16 bits each) starting at the 0^th^ register number (register 0),
+so register numbers 0-1 are mapped onto ClassicLadder's %W variables starting at %W0 ending at %W1.
 
-It's easy to make an off-by-one error as sometimes the modbus elements
-are referenced starting at one rather then 0 (actually by the standard
-that is the way it's supposed to be!) You can use the modbus element
-offset radio button to help with this.
+It's easy to make an off-by-one error as sometimes the modbus elements are referenced starting at one rather then 0 (actually by the standard that is the way it's supposed to be!).
+You can use the modbus element offset radio button to help with this.
 
-The documents for your modbus slave device will tell you how the
-registers are set up- there is no standard way.
+The documents for your modbus slave device will tell you how the registers are set up- there is no standard way.
 
-The SERIAL PORT, PORT SPEED, PAUSE, and DEBUG level are editable for
-changes (when you close the config window values are applied, though
-Radio buttons apply immediately).
+The SERIAL PORT, PORT SPEED, PAUSE, and DEBUG level are editable for changes (when you close the config window values are applied, though Radio buttons apply immediately).
 
-To use the echo function select the echo function and add the slave
-number you wish to test. You don't need to specify any variables.
+To use the echo function select the echo function and add the slave number you wish to test.
+You don't need to specify any variables.
 
-The number 257 will be sent to the slave number you specified and the
-slave should send it back. you will need to have ClassicLadder running
-in a terminal to see the message.
+The number 257 will be sent to the slave number you specified and the slave should send it back.
+You will need to have ClassicLadder running in a terminal to see the message.
 
 == MODBUS Settings
 
 Serial:
 
 * ClassicLadder uses RTU protocol (not ASCII).
 * 8 data bits, No parity is used, and 1 stop bit is also known as 8-N-1.
-* Baud rate must be the same for slave and master. ClassicLadder can
-  only have one baud rate so all the slaves must be set to the same rate.
+* BAUD rate must be the same for slave and master.
```

Comment:
Baud

Reply:
And I was so darn confident. Want to blame some computer magazines from the 80s and thanks go to https://de.wikipedia.org/wiki/Baud .

Reply:
Fixed - also reviewed other .adoc files and s/BAUD/baud/g.

Reply:
Thanks